components/analytics/clickhouse/init.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import sys
+import textwrap
+from argparse import ArgumentParser
+
+import clickhouse_connect
+import clickhouse_connect.driver.client
+
+CLICKHOUSE_HOST_ENV_VAR = "CLICKHOUSE_HOST"
+CLICKHOUSE_PORT_ENV_VAR = "CLICKHOUSE_PORT"
+CLICKHOUSE_DB_ENV_VAR = "CLICKHOUSE_DB"
+CLICKHOUSE_USER_ENV_VAR = "CLICKHOUSE_USER"  # nosec
+CLICKHOUSE_PASSWORD_ENV_VAR = "CLICKHOUSE_PASSWORD"  # nosec
+
+
+def clear_db(client: clickhouse_connect.driver.client.Client):
+    client.query(f"DROP TABLE IF EXISTS {client.database}.events;")
```

Comment:
```suggestion
    client.query("DROP TABLE IF EXISTS events;")
```

`client.database` should be the default database.